dolphinscheduler-api-test/dolphinscheduler-api-test-case/src/test/java/org/apache/dolphinscheduler/api.test/pages/project/WorkFlowDefinitionPage.java

```diff
@@ -0,0 +1,212 @@
+        taskDefinitionRequestData.failRetryInterval("1");
+        taskDefinitionRequestData.failRetryTimes("0");
+        taskDefinitionRequestData.flag("YES");
+        taskDefinitionRequestData.name("echo_123");
+        taskDefinitionRequestData.taskParams(taskParams);
+        taskDefinitionRequestData.taskPriority("MEDIUM");
+        taskDefinitionRequestData.taskType("SHELL");
+        taskDefinitionRequestData.timeout(0);
+        taskDefinitionRequestData.timeoutFlag("CLOSE");
+        taskDefinitionRequestData.timeoutNotifyStrategy("");
+        taskDefinitionRequestData.workerGroup("default");
+
+        ArrayList<Object> taskDefinitionRequestDataList = new ArrayList<>();
+        taskDefinitionRequestDataList.add(taskDefinitionRequestData);
+
+        logger.info("taskDefinitionRequestData：aaaa: %s");
+        logger.info(String.valueOf(taskDefinitionRequestData));
+        logger.info(String.valueOf(taskDefinitionRequestData.getClass()));
+        logger.info(String.valueOf(taskDefinitionRequestDataList));
+        logger.info(String.valueOf(taskDefinitionRequestDataList.getClass()));
+        logger.info("taskDefinitionRequestData：bbbb: %s");
+
+        HashMap<String, Object> conditionParams = new HashMap();
+
+        taskRelationRequestData.name("");
+        taskRelationRequestData.preTaskCode(0);
+        taskRelationRequestData.preTaskVersion(0);
+        taskRelationRequestData.preTaskCode(0);
+        taskRelationRequestData.postTaskCode(genNumId);
+        taskRelationRequestData.conditionType("NONE");
+        taskRelationRequestData.conditionParams(conditionParams);
+        ArrayList<Object> taskRelationRequestDataList = new ArrayList<>();
+        taskRelationRequestDataList.add(taskRelationRequestData);
+
+        HashMap<String, Object> locations = new HashMap();
+        locations.put("taskCode", genNumId);
+        locations.put("x", 33.5);
+        locations.put("y", 38.5);
+        ArrayList<Object> locationsList = new ArrayList<>();
+        locationsList.add(locations);
+
+        workFlowCreateRequestData.locations(JSONUtils.toJsonString(locationsList));
+        workFlowCreateRequestData.taskDefinitionJson(JSONUtils.toJsonString(taskDefinitionRequestDataList));
+        workFlowCreateRequestData.taskRelationJson(JSONUtils.toJsonString(taskRelationRequestDataList));
+        workFlowCreateRequestData.name(workFlowName);
+        workFlowCreateRequestData.tenantCode("admin");
+        workFlowCreateRequestData.executionType("PARALLEL");
+        workFlowCreateRequestData.description("");
+        workFlowCreateRequestData.globalParams("[]");
+        workFlowCreateRequestData.timeout(0);
+
+        headers.put(Constants.SESSION_ID_KEY, sessionId);
+        RequestClient requestClient = new RequestClient();
+        ProjectPage project = new ProjectPage();
+        String projectCode = project.getProjectCode(sessionId, projectName);
+
+        logger.info(String.valueOf(workFlowCreateRequestData));
```

Comment:
Please remove unnessnary log ouput.

Reply:
Thank you for your reminder. Corresponding modifications have been made here